### ISSUE TITLE
UPDATE: Domains and labor categories

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#446]

**Comment by jfredrickson** [comment]
Review: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/jf/domains-labor-categories/oasis-plus/about/domains-naicscodes-scope-labor-categories/

**Comment by vickimcfadden** [comment]
I'll do QA for this one.

**Comment by vickimcfadden** [comment]
@klohman is What’s in scope / out of scope supposed to be H3?

**Comment by klohman** [comment]
@vickimcfadden Yes, "What’s in scope / out of scope" should be a H3 (missed that in the content doc)

**Comment by vickimcfadden** [comment]
@jfredrickson ☝️ could you make this change, and then I think we can consider this done from a design/QA perspective!

**Comment by jfredrickson** [comment]
@vickimcfadden okay, updated! https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/jf/domains-labor-categories/oasis-plus/about/domains-naicscodes-scope-labor-categories/

**Comment by vickimcfadden** [comment]
@PowderD @ekidenda ready for your review https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/jf/domains-labor-categories/oasis-plus/about/domains-naicscodes-scope-labor-categories/

**Comment by PowderD** [comment]
@vickimcfadden This looks good. Especially once all the NAICS pages are changed with the updated tables! TY
